h/authorization.py

```diff
@@ -66,6 +82,48 @@ def is_web_client(request, client_id):
     return client_id == request.registry.web_client.client_id
 
 
+def normalize_authorization_token(request, token):  # bw compat
+    """Convert an Annotator Auth token into a standard JWT Bearer token."""
+    if 'issuedAt' in token:
+        token['iat'] = posix_seconds(iso8601.parse_date(token['issuedAt']))
+        if 'ttl' in token:
+            token['exp'] = token['iat'] + int(token['ttl'])
+            del token['ttl']
```

Comment:
I'd prefer we cloned the `token` object and returned the original instance. Otherwise the caller needs to be aware that the token argument will be modified.